problems.rb

```diff
@@ -0,0 +1,9 @@
+# TODO: Fix this whole function
+def printHi3TimesNTimes(n)
+  for i in 0..n
+    for j in 0..3
+      puts "Hi"
```

Comment:
Consider storing "Hi" in a variable to avoid magic values

Reply:
Done!